src/ui/server/shared-validation/phone-number/index.ts

```diff
@@ -0,0 +1,29 @@
+import { PhoneNumberUtil } from 'google-libphonenumber';
+import generateValidationErrors from '../../helpers/validation';
+import { defaultCountryUK } from '../../constants';
+
+const phoneUtil = PhoneNumberUtil.getInstance();
+
+/**
+ * validates phone number is only from the UK and is a valid phone number
+ * returns validation error if incorrect format
+ * @param {string} phoneNumber
+ * @param {string} fieldId
+ * @param {string} errorMessage
+ * @param {object} errors
+ * @returns {object} updatedErrors
+ */
+const validatePhoneNumber = (phoneNumber: string, fieldId: string, errorMessage: string, errors: object) => {
+  let updatedErrors = errors;
+
+  // if not valid for UK, then returns validation error
+  if (!phoneUtil.isValidNumberForRegion(phoneUtil.parse(phoneNumber, defaultCountryUK), defaultCountryUK)) {
```

Comment:
Can we uppercase the constant please to be consistent with other constants

Reply:
done